### PR TITLE
Abstract Beton

### PR DESCRIPTION
**Pull request by TriXXeD**

### CODE REVIEW

File: abstract.tex [hunk 1]
```diff
@@ -1,7 +1,27 @@
...
+We present an application allowing a network of smartphones to play audio synchronously, effectively increasing volume without degrading quality.
 
-Proin at tellus a elit aliquam bibendum ac nec ipsum. Nulla vitae posuere sem, sit amet dapibus eros. Maecenas pellentesque, velit eu luctus iaculis, ipsum lectus rutrum nisi, eu cursus sapien augue in dui. Fusce eu ligula suscipit, efficitur enim eget, tincidunt velit. Maecenas hendrerit arcu et elit tempus blandit. Duis nec lorem eleifend, faucibus augue vitae, pharetra ligula. Aenean faucibus auctor fermentum.
+\bigskip \noindent
+%Theory stuff : Psycho acoustics : Current Apps(target)
+    %psychoacoustics -interesting application areas - use cases
+    %unsaturated Play Store with dissatisfactory apps
+We investigate the scientific field related to human auditory perception, psychoacoustic effects.
+Our research into psychoacoustics reveals a useful effect, the precedence effect, which amplifies perceived audio by manipulating playback of audio sources.
+We also prove the exsisting solutions on the Play Store to be unsatisfactory, requiring manual trial and error efforts to achieve audible sync.
```
Comment (VirtualSatai) [inline abstract.tex]: sync -> synchronization

File: abstract.tex [hunk 2]
```diff
@@ -1,7 +1,27 @@
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum vitae leo maximus, facilisis turpis id, pellentesque libero. Curabitur blandit diam eget rutrum fringilla. Quisque vestibulum quam sed ligula viverra sagittis.
+%Intro 2
+The ambiance of social events is often filled with backgroud music.
+It is often difficult to achieve desired quality of such music at outdoor ad-hoc events.
+Smartphones, while widespread, lack the hardware to play audio at higher volumes, without distortion.
```
Comment (TriXXeD) [inline abstract.tex]: higher?

**Review by VirtualSatai — APPROVED** [review]

File: abstract.tex [hunk 3]
```diff
@@ -1,7 +1,27 @@
...
+Solutions such as wireless battery powered Bluetooth speakers do exist, but are far less common to have on your person.
+We present an application allowing a network of smartphones to play audio synchronously, effectively increasing volume without degrading quality.
 
-Proin at tellus a elit aliquam bibendum ac nec ipsum. Nulla vitae posuere sem, sit amet dapibus eros. Maecenas pellentesque, velit eu luctus iaculis, ipsum lectus rutrum nisi, eu cursus sapien augue in dui. Fusce eu ligula suscipit, efficitur enim eget, tincidunt velit. Maecenas hendrerit arcu et elit tempus blandit. Duis nec lorem eleifend, faucibus augue vitae, pharetra ligula. Aenean faucibus auctor fermentum.
+\bigskip \noindent
+%Theory stuff : Psycho acoustics : Current Apps(target)
+    %psychoacoustics -interesting application areas - use cases
+    %unsaturated Play Store with dissatisfactory apps
+We investigate the scientific field related to human auditory perception, psychoacoustic effects.
```
Comment (TriXXeD) [inline abstract.tex]: just psychoacoustics

File: abstract.tex (hunk 1 above)
Comment (TriXXeD) [inline abstract.tex]: syncronization

File: abstract.tex (hunk 2 above)
Comment (VirtualSatai) [reply abstract.tex]: 420

**Pull request merged**